index.html

```diff
@@ -0,0 +1,209 @@
+                    <div class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+                    tempor.</div>
+                </div>
+                </div>
+
+                <div class="card_item">
+                <div class="card_inner">
+                    <img src="images/workshop.png" alt="">
+                    <div class="card_title">Workshop</div>
+                    <div class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+                    tempor.</div>
+                </div>
+                </div>
+
+                <div class="card_item">
+                <div class="card_inner">
+                    <img src="images/network.png" alt="">
+                    <div class="card_title">Network</div>
+                    <div class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+                    tempor.</div>
+                </div>
+                </div>
+        </div>  
+		
+                <div class="more-details">
+                <h5>SEE THE WHOLE PROGRAM</h5> 
+                <button class="program-btn" id="join">
+                Join Mo Ibrahim
+                </button>    
+                </div>
+          </div>
+</section>
+
+<div class="speakers">
+        <div class="speaker-heading">
+        <h5>Featured Speakers</h5>
+        <hr>
+        </div>
+
+        <div class="presidentsDiv"></div>
+
+        <button class="showMore" id="show-more">
+        MORE <i class="fa-solid fa-angle-down"></i>
+       </button>
+</div>
+
+<section class="footer">
+    
+         <div class="footer1" id="sponsor">
+             <div class="footer-header">
+             <p>Partner</p>    
+                <hr>
+             </div>
+             <div class="partners">
+                <ul class="flex-items">
+                <li><img src="images/partner1.png" alt="partner1"></li>
+                <li><img src="images/partner2.png" alt="partner2"></li>
+                <li><img src="images/partner3.png" alt="partner3"></li>
+                <li><img src="images/partner4.png" alt="partner4"></li>
+                <li><img src="images/partner5.png" alt="partner5"></li>
+                </ul>
+             </div>  
+         </div>
+
+        <div class="footer2 flex-items">
+            <div class="footer-logo">
+            <img src="images/logo.png" alt="company logo">
+             </div>
+
+            <div class="footer-details">
+            <div class="footer-head">
+                <p>
+            Copyright ©2022 Mo Ibrahim Foundation 
+                </p>
+            </div>
+            <div class="footer-description">
+                <p>
+             The Mo Ibrahim Foundation (MIF) is an African foundation, established in 2006 with one focus: the critical importance of governance and leadership in Africa.
+                </p>
+            </div>
+        </div>
```

Comment:
- [ ] Kindy use the `<footer>` tag in this section instead of a more generic `<div>` tag, this will make your page be accessed faster and help in the SEO of your site.